src/js/menu/acquia_lift.personalize.behaviors.js

```diff
@@ -36,18 +36,13 @@
               }
             }
           } else {
-            var model = Drupal.acquiaLiftUI.factories.MenuFactory.createCampaignModel(obj);
+            var model = new Drupal.acquiaLiftUI.MenuCampaignModel(obj);
             ui.collections.campaigns.add(model);
             addedCampaigns[obj.name] = model;
           }
         });
 
-        // Clear the variations for all page variation campaigns.
-        ui.collections.campaigns.each(function (model) {
-          if (model instanceof Drupal.acquiaLiftUI.MenuCampaignABModel) {
-            model.get('optionSets').resetVariations();
-          }
-        });
+        // Clear the variations for all element variation campaigns.
```

Comment:
This is not clearing but simply merging the option sets from Drupal settings into the Backbone app (and not specific to elements).